### PR TITLE
Expand debugger help message for b command

### PR DESCRIPTION
**Pull request by haltman-at**
Addresses #3268.

I do worry that the help printout is losing its compactness a bit, but...

### CODE REVIEW

File: packages/debug-utils/index.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ const commandReference = {
   "+": "add watch expression (`+:<expr>`)",
   "-": "remove watch expression (-:<expr>)",
   "?": "list existing watch expressions and breakpoints",
-  "b": "add breakpoint",
-  "B": "remove breakpoint",
+  "b": "add breakpoint (can specify line in several ways; see docs)",
```
Comment (gnidan) [inline packages/debug-utils/index.js:28]: it'd be better if we provided some real hint vs. nothing. Maybe this is too much, I don't know:
```suggestion
  "b": "add breakpoint (`b [[<source-file>:]<line-number>]; see docs for more)",
```

Comment (haltman-at) [reply packages/debug-utils/index.js:28]: Oh that's nice; yeah, I'll change that.

**Comment by haltman-at** [comment]
OK, not only did I rewrite the `b` help message as suggested, I also rewrote the `p` message similarly!  Neither of these give the complete syntax, but I think that's OK.  Kind of thinking that maybe `p` should include the `+` syntax though?  IDK.

I suppose `;` could also be similarly rewritten but I don't think it needs it.

**Review by gnidan — APPROVED** [review]
Looks good!

**Pull request merged**